### ISSUE TITLE
Feature request: Add loop and fade options to pygame.mixer.channel.queue()

### ISSUE THREAD

**Issue by OlaKenji** [pygame-community/pygame-ce#3236]
Currently, the pygame.mixer.Channel.queue() method allows queuing a sound to be played immediately after the current sound finishes. However, it does not support specifying loop or fade options for the queued sound. These options are already available in pygame.mixer.Sound.play() and would significantly enhance the functionality and flexibility of queue().

**Comment by AntoineMamou** [comment]
Hello @OlaKenji,

I would like to work on this issue if it's possible

**Comment by AntoineMamou** [comment]
If I understand well the issue, I should add two methods, one for the loop and one for the fade option for the sound that are in the queue.

**Comment by damusss** [comment]
> If I understand well the issue, I should add two methods, one for the loop and one for the fade option for the sound that are in the queue ?

Actually it should not add two methods, but two parameters for the same function aside from the sound being played.

**Comment by AntoineMamou** [comment]
Yes, i agree with you. The params should be add in chan_queue in mixer.c file ?

**Comment by OlaKenji** [comment]
Hi, @AntoineMamou. I am happy that you would like to attempt this. I just wanted to also highlight this issue #3011, in case you happen to come across a solution while digging.

**Comment by Starbuck5** [comment]
I thought this was what @bilhox was trying to solve with https://github.com/pygame-community/pygame-ce/issues/3108
